### ISSUE TITLE
could the 1000 Genomes Project dataset be used to annotate WES data?

### ISSUE THREAD

**Issue by stroke1989** [WGLab/doc-ANNOVAR#177]
Hi,
 I'm a new to annovar, this is a excellent tool. I tried to use annovar to annotate my mutation called from **_WES data_**. could the 1000 Genomes Project dataset be used to annotate mutation called from WES data? the document showed this dataset is used for frequency of variants in **_whole-genome_data** . Hope your response! Appreciate1

**Comment by kaichop** [comment]
Yes it can be used for WES data as well. These are data sets that are
relatively old, so you may want to check gnomad_exome and gnomad_genome
instead.
